packages/vertical-text-block-list/index.test.js

```diff
@@ -7,10 +7,11 @@ const defaultProps = getTestValues(props)
 
 describe('<VerticalTextBlockList />', () => {
   it('should render correctly with default props', () => {
-    render(<VerticalTextBlockList {...defaultProps} />)
-    expect(screen.getByTestId('root').className).toContain(
-      'g-vertical-text-block-list'
+    const className = 'my-text-block-list'
+    const { container } = render(
+      <VerticalTextBlockList {...defaultProps} className={className} />
     )
+    expect(container.firstChild).toHaveClass(className)
```

Comment:
Used `container.firstChild` to assert the `className` on the root element, which allowed the removal of a `data-testid` in the component.